### PR TITLE
add retry to openshift build prometheus ext test initial service access

### PR DESCRIPTION
**Pull request by gabemontero**
see https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_builds/229/display/redirect

@openshift/devex per this morning's email exchange

### CODE REVIEW

File: test/extended/prometheus/prometheus_builds.go [hunk 1]
```diff
@@ -46,7 +46,16 @@ var _ = g.Describe("[Feature:Prometheus][Feature:Builds] Prometheus", func() {
...
+			// allow for some retry, a la prometheus.go and its initial hitting of the metrics endpoint after
+			// instantiating prometheus tempalte
+			var err error
+			for i := 0; i < 30; i++ {
+				err = expectURLStatusCodeExec(ns, execPodName, fmt.Sprintf("https://%s:%d", host, statsPort), 403)
+				if err == nil {
+					break
+				}
+				time.Sleep(time.Second)
+			}
```
Comment (bparees) [inline test/extended/prometheus/prometheus_builds.go]: i am acknowledging that you did not use wait.Poll but lgtm'ing anyway :)

**Comment by gabemontero** [comment]
unit run https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/16684/origin-ci-ut-origin/75/ hit flake https://github.com/openshift/origin/issues/16702

**Pull request merged**